openpype/hosts/flame/plugins/publish/extract_subset_resources.py

```diff
@@ -131,161 +165,157 @@ def process(self, instance):
-                opfapi.export_clip(
-                    export_dir_path, duplclip, preset_path, **kwargs)
-
-                extension = preset_config["ext"]
-
-                # create representation data
-                representation_data = {
-                    "name": unique_name,
-                    "outputName": unique_name,
-                    "ext": extension,
-                    "stagingDir": export_dir_path,
-                    "tags": repre_tags,
-                    "data": {
-                        "colorspace": color_out
-                    },
-                    "load_to_batch_group": load_to_batch_group,
-                    "batch_group_loader_name": batch_group_loader_name
-                }
-
-                # collect all available content of export dir
-                files = os.listdir(export_dir_path)
-
-                # make sure no nested folders inside
-                n_stage_dir, n_files = self._unfolds_nested_folders(
-                    export_dir_path, files, extension)
-
-                # fix representation in case of nested folders
-                if n_stage_dir:
-                    representation_data["stagingDir"] = n_stage_dir
-                    files = n_files
-
-                # add files to represetation but add
-                # imagesequence as list
-                if (
-                    # first check if path in files is not mov extension
-                    [
-                        f for f in files
-                        if os.path.splitext(f)[-1] == ".mov"
-                    ]
-                    # then try if thumbnail is not in unique name
-                    or unique_name == "thumbnail"
-                ):
-                    representation_data["files"] = files.pop()
-                else:
-                    representation_data["files"] = files
-
-                # add frame range
-                if preset_config["representation_add_range"]:
-                    representation_data.update({
-                        "frameStart": frame_start_handle,
-                        "frameEnd": (
-                            frame_start_handle + source_duration_handles),
-                        "fps": instance.data["fps"]
-                    })
-
-                instance.data["representations"].append(representation_data)
-
-                # add review family if found in tags
-                if "review" in repre_tags:
-                    instance.data["families"].append("review")
-
-                self.log.info("Added representation: {}".format(
-                    representation_data))
+            # create preset path
+            preset_orig_xml_path = str(os.path.join(
+                preset_dir, preset_file
+            ))
+
+            preset_path = opfapi.modify_preset_file(
+                preset_orig_xml_path, staging_dir, modify_xml_data)
+
+            # define kwargs based on preset type
+            if "thumbnail" in unique_name:
```

Comment:
Should the `unique_name` be explicitly `"thumbnail"` or any name with `"thumbnail"` in name?
Simplified question: Should this happen if `unique_name` is `"thumbnail_source"`?

Reply:
Yeah there could be a case where `thumbnail_source` can happen - for example, when we support multiple thumbnails in the future.